src/contents/en-gb/doc_xamarin.md

```diff
@@ -0,0 +1,210 @@
+# SAFE Mobile App Tutorial
+
+In this tutorial, we will create a [Xamarin.Forms](https://github.com/xamarin/Xamarin.Forms/) application. Xamarin.Forms is an open source and cross-platform tool which provides a way to quickly build native apps for iOS, Android, Windows and macOS using C# and F#.
+
+To create this app, we will use the [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) NuGet package which exposes the `safe_app` API to connect and interact with the SAFE Network.
+
+Download demo code for this tutorial from [here](https://github.com/maidsafe/safe-getting-started-dotnet/SafeTodoExample).
+
+## Pre-requisites
+
+Before we start working on our first SAFE app, make sure you have the following tools installed to work with this tutorial.
+
+- **Visual Studio with Xamarin**: You need Visual Studio (any edition) to build our first SAFE Network mobile application. Follow the instructions [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/get-started/installation) to download and install visual studio with Xamarin.Forms for your Operating System.
+- **SafeAuthenticator**: Safe Authenticator mobile application will be required to initiate the authentication process with SAFE Network. Please follow installation instruction [here](https://github.com/maidsafe/safe_mobile#Installation).
+
+## Setup basic project
+
+- Create a new Project: Create a new cross-platform mobile app (Xamarin.Forms) project in visual studio.
+- Install MaidSafe.SafeApp package: Install the latest version of [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) package in all projects using NuGet package 
+
+## Using Mock Network
+
+We provide the mock feature with Safe APIs which can be used for fast app development and, also provides a safe space to perform test operations. 
+
+When we use the mock feature in an application, it does not communicate with the live network. Instead, it will interface with a local MockVault file in the system to simulate network operations, which is used to store and retrieve data. Let's see how we can set up and use a mock network.
+
+- Build conditional compilation symbols: Add `SAFE_APP_MOCK` flag for your project in "Properties > Build > Conditional compilation symbols".
+
+Once we set this flag in build settings, a reference to `SafeApp.MockAuthBindings.dll` will be added into the project automatically. It has Additional classes and functions used for mock authentication. We can use any of the following processes for mock authentication based on our requirement.
+
+Following is the example code, we can use for mock authentication.
+We create a mock user account and authenticate using the same without worrying about the safe-browser authentication process.

```

Comment:
always capitals when mentioning the network - 'SAFE' is an acronym and is therefore always written in caps